subsys/jaxrs/src/main/java/org/commonjava/indy/bind/jaxrs/MDCManager.java

```diff
@@ -114,4 +124,17 @@ public void putExtraHeaders( HttpRequest httpRequest )
             }
         } );
     }
+
+    public void putEnvironment()
+    {
+        try
+        {
+            MDC.put( ENVIRONMENT, objectMapper.writeValueAsString( loggingConfig.getEnvars() ) );
```

Comment:
If we are going to send them as a JSON to common logging, to group them in a map would be better. We have other MDC entries like the request-id, etc. If we set the env vars directly, we need to be sure not to override existing keys. We may add new keys in the future and slowly this could be a burden to maintain. But I am not sure how to refer it in logback.xml, like this? %X{evn.hostname} . If we don't need to print it directly to log file, that is no question though.